ci-operator/step-registry/ipi/install/vsphere/registry/ipi-install-vsphere-registry-commands.sh

```diff
@@ -21,7 +21,7 @@ spec:
     requests:
       storage: 10Gi
 EOF
-  oc create -f '${SHARED_DIR}}/registry-pvc.yaml' -n openshift-image-registry &&
+  oc create -f '${SHARED_DIR}/registry-pvc.yaml' -n openshift-image-registry &&
```

Comment:
Beside that, I think here should use `""` instead  of `''`

Reply:
Updated

Reply:
@jianlinliu I have trigger periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-vsphere-ipi-disconnected-registry-pvc-p2-f28 job